sklearn/utils/validation.py

```diff
@@ -212,6 +212,26 @@ def check_consistent_length(*arrays):
                          " samples: %r" % [int(l) for l in lengths])
 
 
+def _make_indexable(iterable):
```

Comment:
we can't have two functions `indexable` and `_make_indexable`, especially not if both are used outside of this module.

Keep `indexable` as the only API and make this local to `indexable`, e.g. `def _check(iterable):`

Reply:
OK but how do I reuse `_check` in `_check_fit_params`. The idea was to avoid code duplication in `indexable` and `_check_fit_params`.